src/main/java/com/gmail/yuramitryahin/entity/Currency.java

```diff
@@ -0,0 +1,22 @@
+package com.gmail.yuramitryahin.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import lombok.Data;
+
+@Data
+@Entity
+public class Currency {
```

Comment:
let's make Currency just an enum - there is no need to store in as a separate entity